tests/compute/dynamic-dispatch-10.slang

```diff
@@ -1,3 +1,5 @@
+//TEST(compute):COMPARE_COMPUTE:-dx11
+//TEST(compute):COMPARE_COMPUTE:-vk
 //TEST(compute):COMPARE_COMPUTE:-cpu -xslang -disable-specialization
```

Comment:
I *think* if we have a cpu definition that will synthesize the cuda test.

Reply:
That it would be good if we had a better way of specifying tests - such that we can just list what it requires, or where it can't work and everything else is synthesized. That's not anything at issue in this PR tho.